catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-converter/src/test/resources/compositecurve-eps26713.xml

```diff
@@ -0,0 +1,42 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<wfs:FeatureCollection xmlns:wfs="http://www.opengis.net/wfs" xmlns:gml="http://www.opengis.net/gml"
+                       xmlns:sf="http://www.openplans.org/spearfish">
+    <gml:featureMembers>
+        <sf:roads gml:id="roads.1">
```

Comment:
❓ what happens if there are multiple "roads" nodes within a "featureMembers" element?  Does that cause multiple metacards to be created?  Do we have a test case for this?

Reply:
Multiple metacards are created.

Reply:
I saw this many times when manually testing against GeoServer. But I'll look into creating a unit test for it as well.

Reply:
unit test has been added

Reply:
thanks 👍